engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/content-view-details.controller.js

```diff
@@ -27,6 +27,7 @@
 angular.module('Bastion.content-views').controller('ContentViewDetailsController',
     ['$scope', 'ContentView', 'ContentViewVersion', 'AggregateTask', 'translate',
     function ($scope, ContentView, ContentViewVersion, AggregateTask, translate) {
+        $scope.contentView = null;
```

Comment:
Ah, that's left over from when I was trying to simply use `ng-show="contentView"`, will remove.